### PR TITLE
Use existing Anilist entry synonyms for matching

### LINKED ISSUE #38: Synonyms are used while trying to add the series to anilist, but not while searching if it's already in the list
First the log to better understand what I'm talking about:
image

Every hour when the script runs, it re-adds BNA to my list with 5 watched episodes. I've had the same problem with some other titles too.

A workaround for this problem is adding the series to the custom mapping, but I'd rather have the script check the synonyms of the series first before deciding to add it to the list.

### PR DESCRIPTION
**Pull request by reconman**
Fixed #38

### CODE REVIEW

**Comment by reconman** [comment]
Now I have nothing more to add, I think.

**Comment by reconman** [comment]
@RickDB I think your Travis integration is broken, the PR checks are not working anymore, but Travis will still build my PR in the background

If you want, I can create another PR for a Github Actions workflow file.

**Comment by RickDB** [comment]
Looks good and merging 👍 

Will check out Travis status and fix it, trying to pick up personal projects again but RL keeps interfering :P

**Comment by reconman** [comment]
You probably have to transfer your project to from travis-ci.org to travis-ci.com and then use the new Github integration app.

For a project of mine that didn't work, so I swapped to Github actions instead. travis-ci.com was not able to find my project anymore.

File: anilist.py [hunk 1]
```diff
@@ -1015,14 +911,15 @@ def match_series_with_seasons(
                         % (plex_title)
                     )
         else:
+            #ignore the Plex year since Plex does not have years for seasons
```
Comment (reconman) [inline anilist.py:914]: Sorry, I forgot a space here. Should be `# ignore the Plex year since Plex does not have years for seasons`

Comment (RickDB) [reply anilist.py:914]: No problem, fixed now :)

**Comment by RickDB** [comment]
Thanks, transferred to travis-ci.com 👍

**Pull request merged**